ietf/submit/tests.py

```diff
@@ -3547,6 +3575,26 @@ def test_cancel_stale_submissions(self):
         self.assertEqual(stale_submission.state_id, 'cancel')
         self.assertEqual(stale_submission.submissionevent_set.count(), 2)
 
+    def test_process_and_accept_uploaded_submission_case(self):
+        """process_and_accept_uploaded_submission should be case-insensitive"""
+        xml, author = submission_file('draft-somebody-test-00', 'draft-somebody-test-00.xml', None, 'test_submission.xml')
+        xml_data = xml.read()
+        xml.close()
+
+        submission = SubmissionFactory(
+            name='draft-somebody-test',
+            rev='00',
+            file_types='.xml',
+            submitter=author.formatted_email().upper(),
```

Comment:
I'd suggest an assertion that `author.formatted_email().upper() != author.formatted_email()` just to be airtight